index.html

```diff
@@ -202,68 +202,48 @@ <h1>Any Questions?</h1>
         <!-- Sponsors -->
         <div class="section sponsors">
             <a class="nav-anchor" id="sponsors"></a>
-            <div class="container">
+            <div class="container" id="sponsor-section">
                 <h1>Sponsors</h1>
+                <!--shifted this up here, similar to pearlhacks and the 2022 hacknc website-->
```

Comment:
You can remove this comment; any context/reasoning like this can just go in the pull request description.